module/Start-EditorServices.ps1

```diff
@@ -123,7 +135,7 @@ function Get-AvailablePort {
 
 # Add BundledModulesPath to $env:PSModulePath
 if ($BundledModulesPath) {
-    $env:PSMODULEPATH = $BundledModulesPath + [System.IO.Path]::PathSeparator + $env:PSMODULEPATH
+    $env:PsModulePath = $BundledModulesPath + [System.IO.Path]::PathSeparator + $env:PsModulePath
```

Comment:
Close - the `s` is capitalized.  Turns out this is important on *nix platforms e.g.:
```
PSModulePath                   /home/hillr/.local/share/powershell/Modules:/usr/local/share/powershell/Modules:/opt/...
```

Reply:
I think this is the only comment left to be addressed, (sorry for adding this on but I really appreciate it @yatli!)

Fix this and this is totally ready to be merged.

Optional: make sure vscode still works 😄

Reply:
@tylerl0706 I'm in a deep profiling session right now so I just quickly edited the file with Github online editor. Will be able to do heavier stuff when I get back home :)